backend/app/services/chat/trigger/unified.py

```
@@ -346,3 +368,42 @@ async def _process_contexts(
+    from app.services.openapi.kb_context import KnowledgeBaseContextCreator
+
+    try:
+        creator = KnowledgeBaseContextCreator(db, user_id)
+        contexts = creator.create_contexts(user_subtask_id, knowledge_base_names)
+        logger.info(
+            "[build_execution_request] Created %d KB contexts from API request "
+            "for subtask %d",
+            len(contexts),
+            user_subtask_id,
+        )
+    except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)